### PR TITLE
Wine API added

### PR DESCRIPTION
**Pull request by aakatev**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] Your additions are ordered alphabetically
- [x] Your submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column should be padded with one space on either side
- [x] You have searched the repository for any relevant issues or pull requests
- [x] Any category you are creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Comment by Tadaboody** [comment]
Oh it's _that_ kind of wine woops

**Comment by aakatev** [comment]
Yeah, it's not Linux wine :sweat_smile:

**Comment by roelofr** [comment]
If you remove the spaces from the checkboxes (the `[x]` thingies), you get an actual checked list.

Also, [squash your commits](https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit).

File: README.md [hunk 1]
```diff
@@ -370,7 +370,8 @@ API | Description | Auth | HTTPS | CORS |
 | [The Report of the Week](https://github.com/andyklimczak/TheReportOfTheWeek-API) | Food & Drink Reviews | No | Yes | Unknown |
 | [TheCocktailDB](https://www.thecocktaildb.com/api.php) | Cocktail Recipes | `apiKey` | Yes | Yes |
 | [TheMealDB](https://www.themealdb.com/api.php) | Meal Recipes | `apiKey` | Yes | Yes |
-| [What's on the menu?](http://nypl.github.io/menus-api/) | NYPL human-transcribed historical menu collection | `apiKey` | No | Unknown |
+| [What's on the menu?](http://nypl.github.io/menus-api/) | NYPL human-transcribed historical menu collection | `apiKey` | No | Unknown |
```
Comment (SuperSandro2000) [inline README.md]:
```suggestion
| [What's on the menu?](http://nypl.github.io/menus-api/) | NYPL human-transcribed historical menu collection | `apiKey` | No | Unknown |
```

**Comment by aakatev** [comment]
> If you remove the spaces from the checkboxes (the `[x]` thingies), you get an actual checked list.
> 
> Also, [squash your commits](https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit).

Thanks, and [x] done!

**Comment by marekdano** [comment]
@aakatev thanks for the contribution!

It looks that the link is broken. 
Please free to open a new PR with the valid link.
Screenshot 2021-03-27 at 20 26 18

**Pull request closed without merging**